assets/js/base/context/container-query-context.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { createContext, useContext } from '@wordpress/element';
+import { useContainerQueries } from '@woocommerce/base-hooks';
+import classNames from 'classnames';
+
+const ContainerQueryContext = createContext( {
+	containerQueryClassName: '',
+	isMobile: false,
+	isSmall: false,
+	isMedium: false,
+	isLarge: false,
+	isLoading: true,
```

Comment:
`isLoading` could conflict with the prop exposed from `LoadingMask` right?  Or were you thinking consumers would just alias it if they need to?

Reply:
In thinking about it, we could just leave this name as is. Consumers can alias.

Reply:
Or maybe `isCalculating` would be more accurate?

Reply:
Yeah... I can't find a better name so they all work for me. But my feeling is that there is nothing really _loading_ or _calculating_ so that might be misleading. Would something like `isPristine` or `isUnrendered` sound good? I'm not sure about them either, though. :confused:

Reply:
`isRendering`?